### PR TITLE
gnudatalanguage: bump to 1.0.0-rc.2

### PR DESCRIPTION
**Pull request by slayoo**
Follow up to #6340.
Not tested (I'm not an osx user).
Thanks in advance for any help in getting this updated!

### CODE REVIEW

File: math/gnudatalanguage/Portfile [hunk 1]
```diff
@@ -63,9 +60,7 @@ conflicts_build             antlr
 
 cmake.out_of_source yes
 
-patchfiles              patch-src-magick_cl.cpp.diff \
-                        patch-src-hdf_fun.cpp.diff
-patchfiles-append       sincos.patch
+patchfiles              patch-src-hdf_fun.cpp.diff
```
Comment (cjones051073) [inline math/gnudatalanguage/Portfile:63]: If some patch files are no longer required, please remember to also delete them as part of this update.

Comment (slayoo) [reply math/gnudatalanguage/Portfile:63]: corrected

**Comment by slayoo** [comment]
Is there any way of debugging the Travis build? The timeout seems fishy - we do use Travis for CI on the gdl repo, we do compile on OSX and the compilation + test execution takes much less than the limit?!

**Comment by mf2k** [comment]
@slayoo : The error is:
`The job exceeded the maximum time limit for jobs, and has been terminated.`

This is "normal" for our Travis builder. It is limited to 50 minutes of build time per process.

**Comment by slayoo** [comment]
Which seems puzzling as GDL compiles on OSX on Travis in less than 300 seconds (https://travis-ci.org/github/gnudatalanguage/gdl/jobs/686488218#L508)

**Comment by reneeotten** [comment]
> Which seems puzzling as GDL compiles on OSX on Travis in less than 300 seconds (https://travis-ci.org/github/gnudatalanguage/gdl/jobs/686488218#L508)

But you are probably not building all the 205 (recursive) dependencies first. If you check the Azure pipelines you'll see that it finishes in 61 minutes (i.e., more than the Travis time-out) and you can see there as well what the build does.

**Comment by slayoo** [comment]
That makes sense - thank you for explanation.

**Comment by cjones051073** [comment]
@slayoo Is there a strong need to update to a release candidate ? Normally we would not do this unless there is a good reason. Is a 1.0.0 final release imminent soon ?

**Comment by slayoo** [comment]
The current macports GDL 0.9.9 dates back to Nov 2018, and there has been a lot of development in the last two years (see release notes: https://github.com/gnudatalanguage/gdl/releases). We intended to make good few release candidates before releasing 1.0.0, in particular hoping from feedback from packagers.

BTW, the Portfile includes ``test.target check``. Are the tests run on the Azure pipeline? I'm trying to locate the logs, but cannot find it.

**Comment by slayoo** [comment]
Anyhow, what is the next step in the workflow?

**Comment by slayoo** [comment]
We have yet another release out (1.0.0-rc3)... but following the experience of #6340 and this issue, let me ask what is a better way to provide support in packaging GDL for macports?

**Review by tenomoto — APPROVED** [review]
Sorry for slow response. I confirmed this changes. I approve this pull request although I am aware of rc3.

**Comment by slayoo** [comment]
thanks Takeshi, how to proceed, how to help?

**Pull request closed without merging**